src/sentry/relay/config/__init__.py

```diff
@@ -162,6 +163,35 @@ def get_dynamic_sampling_config(project: Project) -> Optional[Mapping[str, Any]]
     return None
 
 
+class TransactionNameRule(TypedDict):
+    pattern: str
+    expiry: str
+    scope: Mapping[str, str]
+    redaction: Mapping[str, str]
+
+
+def get_transaction_names_config(project: Project) -> Optional[Sequence[TransactionNameRule]]:
+    if not features.has("organizations:transaction-name-sanitization", project.organization):
+        return None
+
+    cluster_rules = get_sorted_rules(project)
+    if cluster_rules is None or len(cluster_rules) == 0:
```

Comment:
`get_sorted_rules` is guaranteed to return a list, so there is no need to check for `None` here:
```suggestion
    if not cluster_rules:
```